larray/core.py

```diff
@@ -7668,65 +7668,93 @@ def opmethod(self, other):
 
     def __matmul__(self, other):
         """
-        What doc of Numpy matmul says:
-        The behavior depends on the arguments in the following way:
-        * If both arguments are 2-D they are multiplied like conventional matrices.
-        * If either argument is N-D, N > 2, it is treated as a stack of matrices
-          residing in the last two indexes and broadcast accordingly.
-        * If the first argument is 1-D, it is promoted to a matrix by prepending a 1
-          to its dimensions. After matrix multiplication the prepended 1 is removed.
-        * If the second argument is 1-D, it is promoted to a matrix by appending a 1
-          to its dimensions. After matrix multiplication the appended 1 is removed.
-
-        For ND-matrices with N > 2:
-        a @ b[i,j,k,m] = sum(a[i,j,:] * b[k,:,m])
+        Overrides operator @ for matrix multiplication.
+
+        Notes
+        -----
+        Only available with Python >= 3.5
 
         Examples
         --------
-        >>> arr1d = ndtest((3))
-        >>> arr1d
+        >>> if sys.version_info >= (3,5):
```

Comment:
Given the buildbot failing, I wonder if it wouldn't be better to transform those tests to unit tests and either use 
`# doctest: +SKIP` here or even kill these doctests entirely....